src/components/context/CanvasContext.tsx

```diff
@@ -1,11 +1,23 @@
 /* eslint-disable react/jsx-no-constructed-context-values */
-import React, { useState } from 'react';
-import { PropTypes } from 'prop-types';
-import { DEFAULT_NOTE_COLOR } from '../../constants/constants';
+import React, { FC, PropsWithChildren, useState } from 'react';
 
-const CanvasContext = React.createContext();
+import { DEFAULT_NOTE_COLOR } from '../../config/constants';
 
-const CanvasProvider = ({ children }) => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function, @typescript-eslint/no-explicit-any
+const uselessFunc = (_a: any): void => {};
+
+const CanvasContext = React.createContext({
+  noteBeingEditedId: null,
+  setNoteBeingEditedId: uselessFunc,
+  userSetColor: DEFAULT_NOTE_COLOR,
+  setUserSetColor: uselessFunc,
+  highlightNoteBeingEdited: false,
+  setHighlightNoteBeingEdited: uselessFunc,
+  noteBeingTransformedId: null,
+  setNoteBeingTransformedId: uselessFunc,
```

Comment:
I think it would be better to use something like:
```ts
setUserSetColor: (color: string) => null,
```
and adapt the signature to the value you have.